packages/hub/queries/email-card-drop-requests.ts

```diff
@@ -62,6 +62,21 @@ export default class EmailCardDropRequestsQueries {
     return queryResult.rows.map(mapRowToObject)[0];
   }
 
+  // TODO does this cover everything? 🤔
+  async activeReservations() {
+    let db = await this.databaseManager.getClient();
+    const query = `
+      SELECT COUNT(*)
+      FROM  email_card_drop_requests AS t1
+      WHERE
+        requested_at=(SELECT MAX(requested_at) FROM email_card_drop_requests WHERE t1.owner_address=email_card_drop_requests.owner_address)
+        AND claimed_at IS NULL
+    `;
```

Comment:
```
SELECT DISTINCT p1.owner_address FROM email_card_drop_requests p1
LEFT JOIN (SELECT DISTINCT owner_address FROM email_card_drop_requests WHERE claimed_at IS NOT NULL) as p2
ON p1.owner_address = p2.owner_address
WHERE p2.owner_address IS NULL AND p1.requested_at > (now() - interval '60 minutes')
```

This selects unique owner addresses for recent requests that do not have a claim by using a LEFT JOIN to exclude those that do have a claim.

https://stackoverflow.com/a/48646883/9809532

Reply:
thanks, there’s definitely some brokenness in what I had, I’m adjusting the test cases to properly reveal that. But this part seems wrong: `WHERE p2.owner_address IS NULL`… right?

Reply:
Without that, we just get all addresses

Reply:
Wasn't clear earlier - `WHERE p2.owner_address IS NULL` is necessary to exclude the addresses that have claimed. It was not intuitive to me either, but it works!

Reply:
Oh I didn't spot the new change. The new query seems correct! And ya easier to understand